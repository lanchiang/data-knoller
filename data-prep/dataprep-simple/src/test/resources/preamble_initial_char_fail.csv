#preamble
# preamble2
#preamble3
.123 42
.22 23
.21 55
22 22
3 44
32 21
#postamble
#
23 54
#postamble