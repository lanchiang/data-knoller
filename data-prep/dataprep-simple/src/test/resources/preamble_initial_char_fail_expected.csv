.123 42
.22 23
.21 55
22 22
3 44
32 21
23 54